packages/core/admin/_internal/node/createBuildContext.ts

```diff
@@ -170,11 +170,14 @@ const createBuildContext = async ({
 
   const features = strapiInstance.config.get('features', undefined);
 
+  const serveAdminPanel = strapiInstance.config.get('admin.serveAdminPanel', true);
+
   const { bundler = 'webpack', ...restOptions } = options;
 
   const buildContext = {
     appDir,
-    basePath: `${adminPath}/`,
+    // If serveAdminPanel is false, use relative paths for separate deployment
+    basePath: serveAdminPanel ? adminPath : '',
```

Comment:
I haven't tested this yet but given the description of the issue I would expect us to still need the trailing '/' on `adminPath` in the `serveAdminPanel = true` case. Did you test that or have I missed something?

I'll test out the experimental now 🙏🏻

Reply:
I have tested it before opening the PR, but it'd be nice if you could confirm the behaviour with me :D many thanks!